dd-java-agent/agent-mlt/src/main/java/com/datadog/mlt/sampler/ScopeStackCollector.java

```diff
@@ -32,7 +34,9 @@
       ConstantPool<String> stringPool,
       ConstantPool<FrameElement> framePool,
       ConstantPool<FrameSequence> stackPool) {
-    super(new Throwable().getStackTrace(), stringPool, framePool, stackPool);
+    // TODO: for now we are providing empty stacktraces because it's damn too expensive to collect
```

Comment:
I would love this cursing more if it read like this:
```suggestion
    // TODO: for now we are providing empty stacktraces because it's  too damn expensive to collect
```

Reply:
In fact, keep it, it's perfect the way it is.

Reply:
deal 😁